specification/resources/resource-manager/Microsoft.Resources/stable/2015-11-01/resources.json

```diff
@@ -863,7 +877,7 @@
           "200": {
             "description": "",
             "schema": {
-              "$ref": "#/definitions/ResourceListResult"
+              "$ref": "#/definitions/ResourceExpandedListResult"
```

Comment:
>ResourceExpandedListResult [](start = 37, length = 26)

do we want to rename this? customer may or may not specify "$expand"..

Reply:
I've reverted the naming change